src/binding/c/spawn_api.txt

```diff
@@ -0,0 +1,106 @@
+# vim: set ft=c:
+
+MPI_Comm_get_parent:
+    .desc: Return the parent communicator for this process
+    .docnotes: SignalSafe
+{
```

Comment:
notes of `comm_get_parent` is lost.

Reply:
Notes added

Reply:
Also the `SignalSafe` docnotes is removed. We have discussed and agreed that MPI functions should not be declared "SignalSafe" as they are not part of the standard and it is easy to neglect the fact that libc signal unsafe functions is being used somewhere.